seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -33,20 +33,19 @@
       "id": "bd7123c8c441eddfaeb5bdef",
       "title": "Say Hello to HTML Elements",
       "description": [
-        "Welcome to freeCodeCamp's coding challenges. These will walk you through web development step-by-step.",
-        "We recommend you do these in order, but feel free to skip around by clicking the map on the left.",
-        "Also, if you want to save your progress, you should join our community by creating an account. In just a few minutes, you can join our chat room, forum, and even a local study group in your city.",
-        "First, you'll start by building a simple web page using HTML. You can edit <code>code</code> in your <code>text editor</code>, which is embedded into this web page.",
-        "Do you see the code in your text editor that says <code>&#60;h1&#62;Hello&#60;/h1&#62;</code>? That's an HTML <code>element</code>.",
-        "Most HTML elements have an <code>opening tag</code> and a <code>closing tag</code>.",
+        "Welcome to freeCodeCamp's coding challenges. These will walk you through web development, step by step.",
+        "We recommend you do challenges in order but feel free to jump around using the map in the menu.",
+        "To save progress, create an account and join our community. In minutes, you can join our chat room, forum and even a study group in your city.",
+        "You'll start by building a simple web page using HTML. You can write and edit <code>code</code> in the <code>text editor</code> on this web page.",
+        "The code currently in the text editor, <code>&#60;h1&#62;Hello&#60;/h1&#62;</code>, is an HTML <code>element</code>. Most HTML elements have an <code>opening tag</code> and a <code>closing tag</code>.",
         "Opening tags look like this:",
         "<code>&#60;h1&#62;</code>",
         "Closing tags look like this:",
         "<code>&#60;/h1&#62;</code>",
-        "Note that the only difference between opening tags and closing tags is that closing tags have a slash after their opening angle bracket.",
-        "Each challenge has tests that you can run at any time by clicking the \"Run tests\" button. Once you get all tests passing (they change from red to green), you can go to the next coding challenge.",
+        "The only difference between opening and closing tags is the backslash after the opening bracket of a closing tag.",
```

Comment:
Let me know if you would like me to update anything else. Otherwise, I'll start looking at other challenges to see if I can help improve the challenge text.